blocklanguages.php

```diff
@@ -24,44 +24,55 @@
 *  International Registered Trademark & Property of PrestaShop SA
 */
 
-if (!defined('_PS_VERSION_'))
-	exit;
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
 
 use PrestaShop\PrestaShop\Core\Module\WidgetInterface;
 
-class BlockLanguages extends Module implements WidgetInterface
+class blocklanguages extends Module implements WidgetInterface
```

Comment:
I would keep the name with capital letters. I know `php-cs-fixer` correct the class name according to the file name.